portfolio/src/main/webapp/index.html

```diff
@@ -27,6 +26,8 @@ <h2><div id="greeting-container"></div></h2>
       <p>Click here to get a fun fact about me:</p>
       <button onclick="addRandomFunFact()">Here</button>
       <div id="fun-fact-container"></div>
+      <h2>Comments:</h2>
+      <div id="comment-container"></div>
     </div>
   </body>
 </html>
```

Comment:
Please add a newline here, all files should end with a newline.

Reply:
Thanks, fix pushed!